pyproject.toml

```
@@ -1,33 +1,36 @@
-[tool.poetry]
+[project]
 name = "sample-programs"
-version = "2023.05.04"
+version = "2025.03.10"
 description = "Sample Programs in Every Programming Language"
-readme = "README.md"
+readme = ".github/README.md"
+requires-python = ">=3.9,<4.0"
 license = "MIT"
-package-mode = false
+
+dependencies = [
+    "ronbun (>=0.8.1,<0.9)",
+    "glotter2 (>=0.9,<0.10)"
```

Now that `0.10.0` has been released for `glotter2`, this can be done:
```suggestion
    "glotter2 (>=0.10,<0.11)"
```